nipype/interfaces/mrtrix3/reconst.py

```diff
@@ -74,108 +74,55 @@ def _list_outputs(self):
 
 
 class EstimateFODInputSpec(MRTrix3BaseInputSpec):
-    in_file = File(exists=True, argstr='%s', mandatory=True, position=-3,
-                   desc='input diffusion weighted images')
-    response = File(
-        exists=True, argstr='%s', mandatory=True, position=-2,
-        desc=('a text file containing the diffusion-weighted signal response '
-              'function coefficients for a single fibre population'))
-    out_file = File(
-        'fods.mif', argstr='%s', mandatory=True, position=-1,
-        usedefault=True, desc=('the output spherical harmonics coefficients'
-                               ' image'))
+    algorithm = traits.Enum('csd','msmt_csd', argstr='%s', position=-8,
+        mandatory=True, desc='FOD algorithm')
+    dwi_file = File(exists=True, argstr='%s', position=-7,
+        mandatory=True, desc='input DWI image')
+    wm_txt = File(argstr='%s', position=-6,
+        mandatory=True, desc='WM response text file')
+    wm_odf = File('wm.mif', argstr='%s', position=-5, usedefault=True,
+        mandatory=True, desc='output WM ODF')
+    gm_txt = File(argstr='%s', position=-4, desc='GM response text file')
+    gm_odf = File('gm.mif', argstr='%s', position=-3, desc='output GM ODF')
```

Comment:
The new interfaces for estimating the SD responses are now able to estimate different responses for different tissues in case of multi-tissue CSD. However, the related command can be used to estimate  also SD responses without taking into account tissues and therefore only related to white matter:
https://github.com/matteomancini/nipype/blob/e92e11296ccd4e1894ef835d431977bc67834955/nipype/interfaces/mrtrix3/preprocess.py#L23-L25
Setting usedefault=True only for the output related to the white matter response allows to correctly use the command both with canonical and multi-tissues algorithms.